### PR TITLE
Card Live Kicker

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
The Card component is expecting `kickerText` as the string to prefix the headline with. We previously expected this value to be sent by the api but this isn't the case at the moment so this PR adds support for defaulting to 'Live' is the article design type is 'Live'

Screenshot 2020-02-04 at 09 13 48

## Why?
Because we know that, by convention, all Live articles have this prefix so we don't need the api to tell us this

## Link to supporting Trello card
https://trello.com/c/CtQbg9Qj/1139-live-card-kicker

### CODE REVIEW

File: src/web/components/Card/Card.tsx [hunk 1]
```diff
@@ -146,9 +147,21 @@ export const Card = ({
...
-                                        kickerText={kickerText}
-                                        showPulsingDot={showPulsingDot}
-                                        showSlash={showSlash}
+                                        kickerText={
+                                            designType === 'Live'
+                                                ? 'Live'
+                                                : kickerText
+                                        }
+                                        showPulsingDot={
+                                            designType === 'Live' && isLiveBlog
```
Comment (gtrufitt) [inline src/web/components/Card/Card.tsx]: Does designType `live` not imply `isLiveBlog` always true?

File: src/web/components/Card/Card.tsx [hunk 2]
```diff
@@ -146,9 +147,21 @@ export const Card = ({
...
+                                                ? true
+                                                : showPulsingDot
+                                        }
+                                        showSlash={
+                                            designType === 'Live' && isLiveBlog
```
Comment (gtrufitt) [inline src/web/components/Card/Card.tsx]: Nitpick
My preference might be `designType === 'Live' || showSlash` here (if isLiveBlog isn't needed, but actually maybe if it is) - do you prefer the explicit ternary here?

**Comment by paperboyo** [comment]
Minor: are we reserving space for animated dot unlike currentGen? (see [internal discussion](https://chat.google.com/u/0/room/AAAAPTsU18A/GnH6fjO5HLM))

File: src/web/components/Card/Card.tsx (hunk 1 above)
Comment (oliverlloyd) [reply src/web/components/Card/Card.tsx]: I thought you could have designType = Live but not isLiveBlog if you were a live blog but are no longer active but now I'm not so sure.

https://www.theguardian.com/us-news/live/2020/feb/03/democratic-race-starts-in-earnest-with-iowa-caucuses-live-coverage used to be a live blog but now has `designType = Article` and `isLiveBlog = true`

Comment (gtrufitt) [reply src/web/components/Card/Card.tsx]: Yeah, designType on `deadBlogs` is `Article` and you'd only show the pulsing dot and `live` kicker on a live liveblog?

Comment (oliverlloyd) [reply src/web/components/Card/Card.tsx]: So, my logic was off - thanks for highlighting this! It's just designType that matters here so I've made a change to reflect this

File: src/web/components/Card/Card.tsx (hunk 2 above)
Comment (oliverlloyd) [reply src/web/components/Card/Card.tsx]: As per my other comment, isLiveBlog isn't needed so neither is the ternary (stylistically speaking) and we can use a simple conditional

**Comment by oliverlloyd** [comment]
> Minor: are we reserving space for animated dot unlike currentGen? (see [internal discussion](https://chat.google.com/u/0/room/AAAAPTsU18A/GnH6fjO5HLM))

I've never seen any jank when loading this element. Not sure how frontend does things to cause this but in DCR we render everything at the same time and there's no reload that might cause things to jump about

File: src/web/components/Card/Card.tsx (hunk 1 above)
Comment (oliverlloyd) [reply src/web/components/Card/Card.tsx]: Yes, dead blogs are treated like plain Articles (at least in cards anyway). They just have 'as it happened' at the end of the headline

https://www.theguardian.com/us-news/2020/feb/03/americans-shut-out-iowa-caucuses

**Comment by gtrufitt** [comment]
> in DCR we render everything at the same time 

Think this would be a problem on Fronts where containers are rendered server side but not related content

**Pull request merged**